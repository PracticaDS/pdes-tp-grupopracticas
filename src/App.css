.App {
  background: #3c6f84;
}

h1 {
  text-align: center;
  color: orange;
}