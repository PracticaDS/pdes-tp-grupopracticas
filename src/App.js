import React, { Component } from 'react';
import './App.css';
import Fabrica from './components/Fabrica/Fabrica';
import { ToolBox } from './components/ToolBox/ToolBox';
import store from './store';
import { Provider } from 'react-redux';



class App extends Component {

  render() {
    return (
      <Provider store={store}>
        <div className="Container">
          <h1>Revolucion Industrial</h1>
          <div className="App"> 
            <ToolBox /> 
            <Fabrica />
          </div>
        </div>
      </Provider>
    );
  }
}
export default App;