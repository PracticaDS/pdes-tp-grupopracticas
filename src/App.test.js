import React from 'react'
import { shallow } from 'enzyme'
import App from './App'

describe('App', () => {

  it('should render correctly whithout props', () => {
    const component = shallow(<App />)
    expect(component).toMatchSnapshot()
  })
  
})
