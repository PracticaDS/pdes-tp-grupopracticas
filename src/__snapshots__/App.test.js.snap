// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly whithout props 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <div
      className="App"
    >
      <h1>
        Revoluci√≥n Industrial
      </h1>
    </div>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <PrivateRoute
        component={[Function]}
        path="/home"
      />
      <PrivateRoute
        component={[Function]}
        path="/factory"
      />
    </Switch>
  </BrowserRouter>
</Provider>
`;
